config/routes.rb

```diff
@@ -1,11 +1,19 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+
   get 'users/new'
 
- root 'static_pages#home'
+  root 'static_pages#home'
  get  '/help',    to: 'static_pages#help', as: 'help' #get 'static_pages/help'
  get  '/about',   to: 'static_pages#about'
  get  '/contact', to: 'static_pages#contact'
  get  '/signup',  to: 'users#new'
+ post '/signup',  to: 'users#create'
+
+ get    '/login',   to: 'sessions#new'
+ post   '/login',   to: 'sessions#create'
+ delete '/logout',  to: 'sessions#destroy'
```

Comment:
Toàn bộ code ruby dùng nháy đôi nhé
giữa các thành phần cách nhau 1 space thôi nhé